Lists/Lab/RemoveNegativesAndReverse/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace RemoveNegativesAndReverse
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            List<int> numbers = Console.ReadLine()
+                .Split(' ')
+                .Select(int.Parse)
+                .ToList();
+
+            RemovesAndRevers(numbers);
+        }
+
+        static void RemovesAndRevers(List<int> numbers)
```

Comment:
A method name should contain only one action - the method itself should have only a single purpose, only remove or only reverse, but not both